inserts.sql

```diff
@@ -0,0 +1,720 @@
+    (2,	1,	1,	0,	0,	1,	2,	1,	0,	0,	(SELECT id FROM teams WHERE abbreviation = "CLE"),	(Select id FROM players WHERE firstname = "Roberto" AND lastname = "Perez" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 19:00:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "CLE"))),
+    (4,	0,	0,	0,	0,	0,	0,	0,	2,	0,	(SELECT id FROM teams WHERE abbreviation = "SF"),	(Select id FROM players WHERE firstname = "Stephen" AND lastname = "Vogt" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 12:10:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "NYM"))),
+    (3,	0,	1,	0,	0,	0,	0,	0,	0,	0,	(SELECT id FROM teams WHERE abbreviation = "TOR"),	(Select id FROM players WHERE firstname = "Luke" AND lastname = "Maile" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 19:00:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "TOR"))),
+    (4,	0,	0,	0,	0,	0,	0,	0,	1,	0,	(SELECT id FROM teams WHERE abbreviation = "LAA"),	(Select id FROM players WHERE firstname = "Dustin" AND lastname = "Garneau" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 22:07:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "LAA"))),
+    (3,	0,	2,	1,	0,	0,	0,	0,	0,	0,	(SELECT id FROM teams WHERE abbreviation = "ATL"),	(Select id FROM players WHERE firstname = "Brian" AND lastname = "McCann" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 12:35:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "PIT"))),
+    (3,	0,	0,	0,	0,	0,	0,	0,	2,	0,	(SELECT id FROM teams WHERE abbreviation = "WSH"),	(Select id FROM players WHERE firstname = "Yan" AND lastname = "Gomes" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 22:10:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "SD"))),
+    (3,	3,	1,	0,	0,	1,	2,	4,	2,	0,	(SELECT id FROM teams WHERE abbreviation = "HOU"),	(Select id FROM players WHERE firstname = "Robinson" AND lastname = "Chirinos" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 15:40:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "SEA"))),
+    (4,	0,	1,	0,	0,	0,	0,	0,	0,	0,	(SELECT id FROM teams WHERE abbreviation = "NYM"),	(Select id FROM players WHERE firstname = "Tomas" AND lastname = "Nido" AND position = "C"), (SELECT id FROM games WHERE startTime = "2019-06-06 12:10:00" AND homeTeamId = (SELECT id FROM teams WHERE abbreviation = "NYM"))),
+    (3,	0,	0,	0,	0,	0,	1,	0,	2,	0,	(SELECT id FROM teams WHERE abbreviation = "CIN"),	(Select id FROM players WHERE firstname = "Tucker" AND lastname = "Barnhart" AND position = "C"
```

Comment:
Great job getting the subqueries down. Just be careful with your data. There are two Austin Adams who play RP, you'd need team to truly get a unique player. Also you'd need home or away team with games to get a unique game since many games have the same start time.